gradio/_simple_templates/simpledropdown.py

```diff
@@ -21,7 +24,8 @@ def __init__(
         value: str | int | float | Callable | None = None,
         label: str | None = None,
         info: str | None = None,
-        every: float | None = None,
+        every: Timer | None = None,
```

Comment:
we should continue to accept both a `gr.Timer()` for a `float` for convenience and backwards-compatibility
```suggestion
        every: float | Timer | None = None,
```